zokrates_js/build.rs

```diff
@@ -14,7 +14,13 @@ fn export_stdlib() {
     for entry in WalkDir::new(root)
         .into_iter()
         .map(Result::unwrap)
-        .filter(|e| !e.file_type().is_dir())
+        .filter(|e| {
+            !e.file_type().is_dir()
+                && e.file_name()
```

Comment:
https://doc.rust-lang.org/std/path/struct.Path.html#method.extension should be used instead